test/benchmarks/material/button/app.module.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {A11yModule} from '@angular/cdk/a11y';
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {MatButtonModule} from '@angular/material/button';
+
+/**
+ * @title Button benchmark component.
+ */
+@Component({
+  selector: 'app-root',
+  template: `
+    <button id="show" (click)="show()">Show</button>
+    <button id="hide" (click)="hide()">Hide</button>
+
+    <ng-container *ngIf="isVisible">
+			<button mat-raised-button>Basic</button>
```

Comment:
You shouldn't need the `ng-container` ; you can put `ngIf` directly on the button

Reply:
yeah, or alternatively can just use `<ng-template [ngIf]="isVisible>` (if the grouping is intended)

I see that we are doing this in other approved/merged benchmarks too. We should most likely clean this up there too.